packages/data/src/registry.js

```diff
@@ -291,6 +291,10 @@ export function createRegistry( storeConfigs = {}, parent = null ) {
 	// This function will be deprecated as soon as it is no longer internally referenced.
 	//
 	function use( plugin, options ) {
+		if ( ! plugin ) {
+			return;
+		}
```

Comment:
I assume this is a fix for `wp.data.plugins.control`.

Reply:
yep